app/Authentication/AuthServiceProvider.php

```diff
@@ -180,4 +177,15 @@ static function () use ($container): ?Consumer {
             fn () => new Impersonator()
         );
     }
+
+    private function createUitIdV2JwtValidator(DefinitionContainerInterface $container): UitIdV2JwtValidator
+    {
+        $version = $container->get('config')['keycloak']['enabled'] ? 'keycloak' : 'v2';
```

Comment:
Can't we replace the technology specific naming of keycloak by something else?

It makes sense to have a KEYCLOAK_ENABLED toggle. But maybe other config can be less specific

Reply:
I was also thinking about this, but then I thought `v1 and `v2` are more confusing than `uitid`, `auth0`, and `keycloak. It's only needed to know this when bootstrapping, for the rest of the code this is somewhat hidden.